indexes/clock.js

```diff
@@ -55,7 +55,7 @@ module.exports = function (db, opts) {
       opts.keys = false
       opts.values = true
 
-      return pull(index.read(opts), u.Format(keys, values, opts.private === true))
+      return pull(index.read(opts), u.Format(keys, values, opts.private))
```

Comment:
this should be `opts.private === true` so that it's only if exactly `true` not just truthy, which is more likely to happen by accident.

Reply:
Thanks, I've resolved this in `u.Format` instead so that we only have to check it once. See: https://github.com/ssbc/secure-scuttlebutt/pull/228/commits/4f883fce0c3e9f7d3d39a36609f893be9f7d5d43